README.md

```diff
@@ -265,11 +264,40 @@ handshake is significant, so we introduce an algorithm for "request
 skipping" that avoids sending unnecessary requests, and saves a lot of
 bandwidth compared to just requesting all feeds each connection.
 
+## Related work
+
+[Brisa] also describes a broadcast protocal that at first glace looks
+very close to the [EBT paper]. It is modelled using two components:
+tree construction/maintenance and peer sampling. Peer sampling is in
+SSB terminology where [SSB conn] is used. Brisa uses [HyParView],
+written by the same authors as the [EBT paper] for peer
+sampling. Compared to EBT, Brisa does not depend on lazy mode between
+peers where only the sequence information is maintained, instead it
+depends on HyParView to detect failures. This has the advantage that
+it does not need a timer, that is highly latency sensitive. It also
+has a nice property in how messages are disseminated in that they are
+piggybacked with information about the tree that allows the parent
+selection to make better choices as it has a better view of the
+network. The sequence numbers are an important part of the protocol
+implemented here because they, as described earlier, are used to
+ensure that messages are disseminated in a eventually consistent
+manor.
+
+
 ## TODO
 
 * handle models where it's okay to have gaps in a log (as with classic
-  [insecure scuttlebutt](https://github.com/dominictarr/scuttlebutt)
+  [insecure scuttlebutt]
 
 ## License
 
 MIT
+
+[EBT paper]: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.190.3504
+[Brisa]: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.360.1724
+[HParView]: http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.190.3289
```

Comment:
Small typo. Missing an Y in the name thing

Reply:
Good find. Thanks!